### PR TITLE
Tooltips with react-intl

### LINKED ISSUE #4551: Tooltip for Tabbar does not show up (only accounts show)

### PR DESCRIPTION
**Pull request by jacogr**
(Split from https://github.com/ethcore/parity/pull/4539)

- Allow react-intl messages with `~/ui/Tooltips/Tooltip`
- Change text elements to `FormattedMessage` where Tooltips are constructed
- Fix main Application Tabbar Tooltip not showing (Closes #4551)
- Simple smoke-tests for Tooltips/Tooltip

### CODE REVIEW

File: js/src/ui/Tooltips/Tooltip/tooltip.js [hunk 1]
```diff
@@ -30,15 +31,15 @@ let tooltipId = 0;
...
 class Tooltip extends Component {
   static propTypes = {
-    title: PropTypes.string,
-    text: PropTypes.string,
-    right: PropTypes.bool,
+    className: PropTypes.string,
     currentId: PropTypes.number,
     maxId: PropTypes.number,
-    className: PropTypes.string,
+    right: PropTypes.bool,
```
Comment (ngotchac) [inline js/src/ui/Tooltips/Tooltip/tooltip.js]: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md

Should we ? (PS, `r` is after `o` :laughing:)

Comment (jacogr) [inline js/src/ui/Tooltips/Tooltip/tooltip.js]: Funnily enough, I played with that some weeks ago - it does some nice stuff, e.g. required first, then normal, then handlers (`on`Something).

Comment (ngotchac) [inline js/src/ui/Tooltips/Tooltip/tooltip.js]: So shouldn't we use it ?

Comment (jacogr) [inline js/src/ui/Tooltips/Tooltip/tooltip.js]: We should, the only reason I have not pulled it in as part of one of my eslint updates (yet) is that we are basically wrong in 70% of the cases. So lot of small little niggly updates, just didn't have the time to fiddle.

Medium-term, I would love to.

**Pull request merged**